chain/client/src/test_utils/mod.rs

```diff
@@ -0,0 +1,17 @@
+use near_epoch_manager::RngSeed;
+
+pub mod block_stats;
+pub mod client;
+pub mod peer_manager_mock;
+pub mod setup;
+pub mod test_env;
+pub mod test_env_builder;
+
+pub use block_stats::*;
+pub use client::*;
+pub use peer_manager_mock::*;
+pub use setup::*;
+pub use test_env::*;
+pub use test_env_builder::*;
+
+const TEST_SEED: RngSeed = [3; 32];
```

Comment:
nit: probably can be moved to `setup.rs`

Reply:
It's used in `test_env.rs` and `test_env_builder.rs` as well.

Reply:
Right, will shift there!